templates/package/images_form.html

```diff
@@ -87,6 +81,64 @@ <h2>{{ package.name }}'s images</h2>
         });
     });
 
+    const client = filestack.init('AnCMyPYeSUUEf6IM4xBdwz');
+    const options = {
+        "container": ".picker-content",
+        "accept": ["image/png", "image/jpg"],
+        "maxFiles": 5,
+        "maxSize": 1024*1024*5,
+        "fromSources": ["local_file_system"],
+        "uploadInBackground": false,
+        "onFileUploadFinished": fillForm,
+        "transformations": {
+        "crop": true,
+        "circle": true,
+        "rotate": true
+        }
+    };
+     const fileStackOpen = () => {
+         client.picker(options).open();
+     }
+
+     function fillForm(response){
+            document.getElementById("formset-add-new-form-btn").click();
+            let form_id = document.getElementById("id_form-TOTAL_FORMS").value - 1;
+            let form_url = document.getElementById("id_form-" + form_id + "-url")
+            let form_img = document.getElementById("uploaded-"+ form_id +"-img")
+            form_url.value = response.url;
+            form_img.src = response.url;
+            let parentForm = form_url.parentNode;
+            let formDescr = parentForm.firstChild;
+            formDescr.innerText = "Upload";
+     }
+
+     function deleteImg(clickedSpan) {
+        let deleteImgUrl = clickedSpan.nextSibling.src;
+        document.getElementById("formset-add-new-form-btn").click();
+        let form_id = document.getElementById("id_form-TOTAL_FORMS").value - 1;
+        let imgForm = document.getElementById("uploaded-"+ form_id +"-img");
+        let urlForm = document.getElementById("id_form-"+ form_id +"-url");
+        let parentForm = imgForm.parentNode;
+        console.log(parentForm);
+        imgForm.src = deleteImgUrl;
+        urlForm.value = deleteImgUrl;
+        document.getElementById("id_form-"+ form_id +"-DELETE").checked = true;
+        let formDescr = parentForm.firstChild;
+        formDescr.innerText = "Delete";
+     }
+
+     function returnForm(clickedSpan){
+         let formElement = clickedSpan.parentNode;
+         let deleteCheck = formElement.childNodes[2];
+         let image = formElement.childNodes[5]
+         if(deleteCheck.checked == true){
+            let allUploadedImages = document.getElementById("img-entries").childNodes;
+            let hiddenImages = Object.values(allUploadedImages).filter(imgForm => imgForm.style.display == 'none');
+            let findedElement = hiddenImages.find(element => image.src.includes(element.lastChild.src));
+            findedElement.style.display = "";
+         }
+     }
+
```

Comment:
how much of those variables defined with `let` in 3 functions above , could be actually defined with `const`?